lib/flutrans.dart

```diff
@@ -36,21 +36,13 @@ class Flutrans {
     finishCallback = callback;
   }
 
-  Future<void> init(String clientId, String url,
-      {String env = 'production'}) async {
-    await _channel.invokeMethod("init", {
-      "client_key": clientId,
-      "base_url": url,
-      "env": env,
-    });
+  Future<void> init(String clientId, String url) async {
```

Comment:
env works on IOS, not sure why you remove it?

Reply:
my mistakes, I accidentally remove that lines.